inference-engine/src/inference_engine/ie_network_reader.cpp

```diff
@@ -147,7 +147,9 @@ void assertIfIRv7LikeModel(std::istream & modelStream) {
         }
     }
 
-    THROW_IE_EXCEPTION << "IR v" << irVersion << " is deprecated. Please, migrate to IR v10 version";
+    THROW_IE_EXCEPTION << "The support of IR v" << irVersion <<  " has been removed from the product. "
+        "Please, convert the original model using the Model Optimizer which comes with this "
+        "version of the Inference Engine to generate supported IR version.";
```

Comment:
Maybe we should change IE to OpenVINO because OpenVINO is a tool kit which includes Inference Engine and Model Optimizer.

 version of the Inference Engine -> version of the OpenVINO?

Reply:
I don't mind. Fixed